.github/workflows/check_suite.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check Enforcer
+
+on:
+  check_suite:
+    types: [completed]
+
+jobs:
+  check-enforcer:
+    name: Evaluate Check Suite
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Run
+        env:
+          GITHUB_PAYLOAD: ${{ toJson(github.event) }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "Sparse checkout azure-sdk-tools..."
+          git clone --no-checkout --filter=tree:0 https://github.com/benbp/azure-sdk-tools .
```

Comment:
```suggestion
          git clone --no-checkout --filter=tree:0 https://github.com/azure/azure-sdk-tools .
```

Reply:
Full disclosure I get why it's like this for testing purposes. Just FYI in case you forget.